### PR TITLE
Tweaked tests

### PR DESCRIPTION
**Pull request by Lagicrus**
Tweaked the divisors test as well as the compare array test.
The reason behind the divisors test is that we do not test to see if it actually returns something and if it does not return anything it counts as an `undefined`. Which when ignored and used in the compare function array it can either pass due to JS and its wonderful logic about undefined being in arrays. Also, checking here makes the error not occur later within the compare array function checker.

As for the checks for the array comparing it is much the same.
If the user does not check the length of the arrays then it is possible to pass (given a certain setup*) the test.
The blank compare is just a test on how the user compares because in my mind their function might fail if two blanks are inserted.
The final test is to see how they deal with a blank list vs a filled list as with my testing if the array they iterate over is empty it skips the for loop and returns true as no fail statements were encounted.

tldr; This is a set of tweaks to the pre-existing test setup that is designed to avoid the breaks and "cheats" that I found in issues #7 and #8 where passing blank "works" because the user designed the compare statement in a certain way.

* The setup I am referring to here is a "simple" for loop that iterates over the value within the array and checks to see if the other array includes it.
```js
    for (const value in array_1) {
        if (!(array_2.includes(array_1[value]))){
            return false;
        }
        if (!(array_1[value] === array_2[value])) {
            return false;
        }
    }
    return true```

### CODE REVIEW

**Comment by ear1grey** [comment]
Thanks for the contrbution.  These additional tests have (at last!) been hand stitched into a `test.js` following a big overhaul today (but for simplicity not merged because the index.html needs to stay out of the repo, and the structural changes are large, but herein recognition) :-)
